spec/commands/v2/publish_spec.rb

```diff
@@ -341,7 +347,13 @@
           payload[:update_type] = "minor"
           ChangeNote.create(edition: draft_item)
         end
+
         it "deletes associated ChangeNote records" do
+          FactoryGirl.create(:live_edition,
+            document: draft_item.document,
+            base_path: base_path,
+          )
+
```

Comment:
Is this and the above item to ensure we have a previous item for a minor type? If so is there any chance we could refactor things around so this is within a before block for a context so we can try keep the `it` blocks related focused on the test?